applications/tune_psf.py

```diff
@@ -78,12 +78,12 @@
 from simtools import io_handler, visualize
 from simtools.model.telescope_model import TelescopeModel
 from simtools.ray_tracing import RayTracing
```

Comment:
It seems these two modules haven't changed: TelescopeModel and RayTracing.

Reply:
These are class names. Class names should stay in CamelCase according to [PEP8](https://peps.python.org/pep-0008/#class-names)